core/capabilities/webapi/target/connector_handler.go

```diff
@@ -72,6 +72,7 @@ func (c *ConnectorHandler) HandleSingleNodeRequest(ctx context.Context, messageI
 
 	select {
 	case resp := <-ch:
+		l.Debugw("received response from gateway")
```

Comment:
nit: maybe more info?

Reply:
will leave this as is because this log contains `messageID`